### DISCUSSION TITLE
Extended script for Hager EHZ363, Apator Norax 3D (SML)

### DISCUSSION THREAD

**Discussion by Dropdragger** [arendst/Tasmota#19907] [Show and tell]
I built an optical interface to the e-meter. The e-meter I installed is of the Norax3D type and comes from Apartor. With the script available on this website, the reading worked without any problems. But I was annoyed that I could only read the overall performance. The values for the individual phases were only output in the form of voltage, current and phase. The total performance calculated from this only corresponds very imprecisely to the value given.

I then analyzed the raw data stream and found that additional values were output in it. In particular, I also found the power values of the individual phases here. I have added this to the existing script and have now also these values included.

Here are the corresponding screenshots:

![Script](https://github.com/arendst/Tasmota/assets/49004138/d47c87be-6198-43aa-801c-7849e6ca8e50)

![Display](https://github.com/arendst/Tasmota/assets/49004138/a7d667dc-6b91-460c-b0e8-a78f5046c627)

**Comment by hubtoe** [comment]
Sehr interessant, aber leider bekomme ich keine Leistungsdaten auf der einzelnen Phase angezeigt
